README.md

```diff
@@ -1,90 +1,141 @@
+-   [Visualizations with
+    `DataCombined`](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/data-combined-plotting.html)
+
+# Installation {#installation}
+
+The **ospsuite** package is compatible with R version 3.6.x **AND**
+4.x.x and can be used on [Windows](#on-windows) and [Linux](on-linux)
+operating systems.
+
+`ospsuite` depends on three packages that are not available on CRAN and
+need to be installed separately:
+
+-   `rClr`
+
+    -   [For R
+        4.x.x](https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.2/rClr_0.9.2.zip)
+
+    -   [For R
+        3.6.x](https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1-R3/rClr_0.9.1.zip)
+
+-   [`ospsuite.utils`](https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/releases/latest)
+
+-   [`tlf`](https://github.com/Open-Systems-Pharmacology/TLF-Library/releases/latest)
+
+## On Windows {#on-windows}
+
+### Pre-requisites
+
+The package requires the Visual C++ Runtime that is installed with OSPS
+and can be manually downloaded
+[here](https://aka.ms/vs/16/release/vc_redist.x64.exe).
+
+### From GitHub (recommended)
+
+The latest released version of the package can be installed from GitHub
+using the `{remotes}` package. The code below will download and install
+all the required dependencies.
+
+``` r
+install.packages("remotes")
+rClrPath <- if (R.Version()$major >= 4) {
+  "https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.2/rClr_0.9.2.zip"
+  } else {
+    "https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1-R3/rClr_0.9.1.zip"
+  }
+  
+install.packages(rClrPath,
+                 repos = NULL,
+                 type = "binary"
+)
+
+remotes::install_github("Open-Systems-Pharmacology/OSPSuite.RUtils@*release")
+remotes::install_github("Open-Systems-Pharmacology/TLF-Library@*release")
+remotes::install_github("Open-Systems-Pharmacology/OSPSuite-R@*release")
+```
 
-## Under Windows
+### From package archive files
 
-The release version of the package comes as a binary `*.zip` and can be downloaded from [here](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/releases).
+The release version of the package comes as a binary `*.zip` and can be
+downloaded from
+[here](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/releases).
+You will also have to download and manually install the packages `rClr`,
+`ospsuite-utils`, and `TLF`, and install their dependencies from CRAN.
 
-If you use [RStudio IDE](https://www.rstudio.com/), you can use the *Install* option in the *Packages* pane and select the option *Install from -> Package Archive File* to install a package from binary `*.zip` files.
+If you use [RStudio IDE](https://www.rstudio.com/), you can use the
+*Install* option in the *Packages* pane and select the option *Install
+from -\> Package Archive File* to install a package from binary `*.zip`
+files.
 
 To install manually, follow these instructions:
 
-```r
+``` r
```

Comment:
@Felixmil I was thinking about having a script here (as a "hidden" R-chunk, or is it only possible in vignettes?) that parses the `DESCRIPTION` files and offers an `install.packages()` line with all required packages from CRAN.